apps/docs/src/getting-started/features/maintenance-mode.md

```diff
@@ -80,3 +80,33 @@ const isMaintenanceMode = computed(() => {
   <div v-if="isMaintenanceMode">Maintenance Mode Page Content</div>
 </template>
 ```
+
+### IP Whitelisting
+
+This document provides a step-by-step guide on how to add the possibility for whitelisting in the Frontends app.
+Whitelisting allows specific users or IP addresses to bypass certain restrictions or maintenance modes, ensuring 
+they have access to the application even when it is otherwise restricted.
+ 
+
+The solution involves adding a server middleware that checks whether maintenance mode is enabled. If maintenance mode is active, SSR (Server-Side Rendering) mode will be off. This ensures that the backend IP is omitted, and CRS will take the role to display the maintenance page.
+
+```ts
+// frontends/templates/vue-demo-store/server/middleware/maintenance.ts
```

Comment:
let's add this as working code in some example and use AutoMD to generate code in docs, thanks to this we'll know if some code examples are obsolete

Reply:
:arrow_down: use the resolve conversation button :+1: